hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/net/NodeSchemaLoader.java

```diff
@@ -227,7 +229,7 @@ private NodeSchemaLoadResult loadSchemaFromYaml(InputStream schemaFile) {
     NodeSchemaLoadResult finalSchema;
 
     try {
-      Yaml yaml = new Yaml();
+      Yaml yaml = new Yaml(new SafeConstructor(new LoaderOptions()));
```

Comment:
Looks like this causes test failures:

```
 Tests run: 5, Failures: 4, Errors: 0, Skipped: 0, Time elapsed: 0.268 s <<< FAILURE! - in org.apache.hadoop.hdds.scm.net.TestYamlSchemaLoader
```

https://github.com/rohit-kb/ozone/actions/runs/3882072979/jobs/6621790406#step:6:584

Can you please check?

Reply:
thanks for the review, looking into it